ampd/src/broadcaster/mod.rs

```diff
@@ -87,7 +87,7 @@ impl Default for Config {
             chain_id: "axelar-dojo-1".parse().unwrap(),
             tx_fetch_interval: Duration::from_millis(500),
             tx_fetch_max_retries: 10,
-            gas_adjustment: 1.0,
+            gas_adjustment: 1.1,
```

Comment:
```suggestion
            gas_adjustment: 1.2,
```
i'd suggest 1.2, 1.1 has issues as well sometimes